src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -5,26 +5,33 @@ import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
 import PropTypes from 'prop-types';
 import {withNavigation} from '@react-navigation/compat';
-import styles from '../../../styles/styles';
-import SidebarLinks from './SidebarLinks';
-import PopoverMenu from '../../../components/PopoverMenu';
-import FAB from '../../../components/FAB';
-import ScreenWrapper from '../../../components/ScreenWrapper';
-import Navigation from '../../../libs/Navigation/Navigation';
-import ROUTES from '../../../ROUTES';
-import Timing from '../../../libs/actions/Timing';
-import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
-import CONST from '../../../CONST';
-import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
-import compose from '../../../libs/compose';
-import * as Expensicons from '../../../components/Icon/Expensicons';
-import Permissions from '../../../libs/Permissions';
-import ONYXKEYS from '../../../ONYXKEYS';
-import * as Policy from '../../../libs/actions/Policy';
-import Performance from '../../../libs/Performance';
-import * as Welcome from '../../../libs/actions/Welcome';
+import styles from '../../../../styles/styles';
+import SidebarLinks from '../SidebarLinks';
+import PopoverMenu from '../../../../components/PopoverMenu';
+import FAB from '../../../../components/FAB';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ROUTES from '../../../../ROUTES';
+import Timing from '../../../../libs/actions/Timing';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../../components/withWindowDimensions';
+import CONST from '../../../../CONST';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import compose from '../../../../libs/compose';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import Permissions from '../../../../libs/Permissions';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import * as Policy from '../../../../libs/actions/Policy';
+import Performance from '../../../../libs/Performance';
+import * as Welcome from '../../../../libs/actions/Welcome';
 
 const propTypes = {
+
+    /* Create Listener callback */
+    afterShowCreateMenu: PropTypes.func,
```

Comment:
I know I suggested this name, but generally, I avoid `before` or `after` naming conventions, and rather use `onHide`, etc. Let me think if we can name it better. Do let me know if you have better suggestions.